src/pages/Login/Login.jsx

```diff
@@ -7,7 +7,9 @@ import './Login.scss'
 const Login = () => (
   <div className="container">
     <div className="field">
+      <h1 className="welcome__login">Welcome to AnimeFlix</h1>
```

Comment:
To keep the BEM, we can change the name of those classes. Thinking about a block, will be `welcome`, so we can change the `container` to be `welcome`. Then, we use `field` when we have inputs inside. In this case, we can keep it like `welcome__container` maybe? And for the title, `welcome__title`